src/client/lib/utils.js

```diff
@@ -82,7 +82,7 @@ var run = (g, cb) => {
  * @param  {Generator} gen
  * @return {Promise}
  */
-export var async = (gen) => {
+export let async = (gen) => {
```

Comment:
'export' is only available in ES6 (use esnext option).<br>'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).